WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -122,6 +130,25 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
     }
     
     internal func capturePhoto() {
+        // before capture photo, judge `photoOutput` connection input port have valid video media type, if not report a error
```

Comment:
We don't need the comment here either I don't think.

Reply:
yep, code is self explanatory.